native/packages/react-native-bpk-styles/readme.md

```diff
@@ -8,31 +8,94 @@
 npm install react-native-bpk-styles --save-dev
 ```
 
-## Usage
+## Shadows
+
+The package includes definitions for the Backpack shadows. These are available as objects that can be spread in style definitions. These are iOS only and need to be complimented with an appropriate elevation for Android.
+
+
+### Usage
 
 ```javascript
 import React from 'react';
 import { StyleSheet } from 'react-native';
-import { colorWhite, borderRadiusSm } from 'bpk-tokens/tokens/base.react.native';
+import {
+  colorWhite,
+  borderRadiusSm,
+  elevationBase,
+  elevationLg
+} from 'bpk-tokens/tokens/base.react.native';
 import { shadows } from 'react-native-bpk-styles';
 
 const styles = StyleSheet.create({
   component: {
     backgroundColor: colorWhite,
     borderRadius: borderRadiusSm,
-    ...shadows.base(),
+    ...Platform.select({
+      ios: shadows.base(),
+      android: elevationBase
+    })
   },
-  componentFocused: shadows.large(),
+  componentFocused: Platform.select({
+    ios: shadows.large(),
+    android: elevationLg
+  })
 });
 
-const MyComponent = (props) => {
+const MyComponent = props => {
   const { focused, children } = props;
   return (
     <View style={[styles.component, focused && styles.componentFocused]}>
-      { children }
+      {children}
     </View>
   );
 };
 
 export default MyComponent;
 ```
+
+## Gradients
+
+The package includes definitions for the Backpack gradients. These are available as objects that can be spread in the props of a `LinearGradient` from `react-native-linear-gradient`.
+
+### Usage
+
+```javascript
+import React from "react";
+import { View } from "react-native-linear-gradient";
+import LinearGradient from "react-native-linear-gradient";
+import {
+  colorWhite,
+  borderRadiusSm,
+  elevationBase,
+  elevationLg
+} from "bpk-tokens/tokens/base.react.native";
+import { gradients } from "react-native-bpk-styles";
+
+const MyComponent = props => {
+  return (
+  <View>
+    <LinearGradient {...gradients.primary()}>
+      {children}
+    </LinearGradient>
+    <LinearGradient {...gradients.primary(gradients.ANGLES.down)}>
+      {children}
+    </LinearGradient>
+  </View>
+  );
+};
+
+export default MyComponent;
+```
+
+### API
+
+The `gradients.primary` function takes an optional argument indicating the direction of the gradient. The valid directions are avaialble in `gradients.ANGLES` and are:
+
++ `ANGLES.down`
```

Comment:
I wasn't aware we had different gradient directions?

Reply:
In web we only have one, see https://backpack.github.io/sassdoc/#colors-variable-bpk-primary-gradient

Reply:
The topLeft(135deb) one is the default but me and @jamesf3rguson agreed to support other directions too